### DISCUSSION TITLE
PostgREST-GO practical tutorial and guide

### DISCUSSION THREAD

**Discussion by mrsse** [PostgREST/postgrest#3833] [Q&A] [answered]
appreciate to guide me if there is any released official tutorial and guide on how to use postgREST-GO client package as the documentation is not very clear for beginners in golang.

**Comment by wolfgangwalther** [comment, marked as answer]
The postgrest-go repo is [here](https://github.com/supabase-community/postgrest-go) and has a link to the [docs](https://pkg.go.dev/github.com/supabase-community/postgrest-go) in its README.

We don't support any of the client libraries in any official way, so you're better off to ask in their repo.
